docs/BreakingChanges/067 Deserialization of objects across appdomains can fail.md

```diff
@@ -0,0 +1,22 @@
+## 67: Deserialization of objects across appdomains can fail
```

Comment:
{
  "Id": "67",
  "Title": "Deserialization of objects across appdomains can fail",
  "Details": "In some cases, when an app uses two or more app domains with different application bases, trying to deserialize objects in the logical call context across app domains throws an exception.",
  "Suggestion": "See [Mitigation: Deserialization of Objects Across App Domains](https://msdn.microsoft.com/en-us/library/dn458353(v=vs.110).aspx)",
  "Link": "https://msdn.microsoft.com/en-us/library/dn458360(v=vs.110).aspx",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5.1",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [ ] Not detectable via API analysis